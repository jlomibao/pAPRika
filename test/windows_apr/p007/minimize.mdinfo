

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1222E+02     4.6847E-01     1.6092E+00     H10        35

 BOND    =        1.1172  ANGLE   =      107.7367  DIHED      =       51.4805
 VDWAALS =      -48.9567  EEL     =     1337.2047  EGB        =     -133.8590
 1-4 VDW =        5.2202  1-4 EEL =    -2132.1718  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================
