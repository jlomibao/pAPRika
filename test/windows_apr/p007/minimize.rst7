defa
  125
  -0.5217339   4.9756640   1.8355848   1.0157604   4.9105465   1.8355196
  -0.8854786   4.2788179   0.6087443  -0.8853379   4.2790727   3.0626210
  -0.9016121   6.0038330   1.8355126   1.4812215   5.9029098   1.8354101
   1.3195224   4.1853004   0.6089266   1.3196004   4.1855565   3.0622539
   2.6867685   4.0831510   0.1140544   2.6867161   4.0828887   3.5572286
   0.2014468   3.8598812   3.7842956   0.2012262   3.8584629  -0.1124092
   0.1805438   3.3535085   4.8928155   0.1800717   3.3491237  -1.2199057
   2.6632163   4.0467198  -0.9811883   3.2458854   4.9654716   0.4462377
   2.6628222   4.0457791   4.6524411   3.2460502   4.9652667   3.2255636
  -4.6354270   1.9607215   1.8359929  -3.8096146   3.2592677   1.8358961
  -4.2143484   1.2970534   0.6090678  -4.2140874   1.2970499   3.0628394
  -5.7156950   2.1463555   1.8360788  -4.4358641   4.1588512   1.8359619
  -3.0301004   3.1593200   0.6089175  -3.0298701   3.1592592   3.0627726
  -2.2565023   4.2912447   0.1139237  -2.2561849   4.2910703   3.5577993
  -3.3065502   2.0275780   3.7842663  -3.3065672   2.0273950  -0.1121487
  -2.8770733   1.7549562   4.8920424  -2.8758984   1.7539298  -1.2194771
  -2.2362777   4.2523329  -0.9813003  -2.7399119   5.2173196   0.4457907
  -2.2354571   4.2515262   4.6529912  -2.7398158   5.2172285   3.2264877
  -4.0830460  -3.1096672   1.8358370  -4.7945921  -1.7451322   1.8359154
  -3.2977758  -3.0768203   0.6088935  -3.2975627  -3.0767757   3.0626460
  -4.7840117  -3.9523332   1.8358830  -5.8867915  -1.8375415   1.8359900
  -4.3182061  -1.1199848   0.6089645  -4.3179498  -1.1200118   3.0627979
  -4.9112275   0.1162759   0.1142670  -4.9106749   0.1162157   3.5577926
  -3.4763352  -1.9256347   3.7841675  -3.4762743  -1.9254077  -0.1122164
  -3.0254433  -1.6901620   4.8919509  -3.0240682  -1.6891087  -1.2195152
  -4.8675772   0.1144716  -0.9809636  -5.9548681   0.1611091   0.4463083
  -4.8663544   0.1143033   4.6529962  -5.9544576   0.1610621   3.2262181
   0.5850572  -5.1649837   1.8355222  -0.9524670  -5.0997426   1.8356240
   0.9488432  -4.4682192   0.6086015   0.9489511  -4.4681605   3.0623803
   0.9648910  -6.1931691   1.8354899  -1.4180220  -6.0920618   1.8356626
  -1.2560924  -4.3746847   0.6087354  -1.2558866  -4.3745872   3.0625218
  -2.6233415  -4.2706186   0.1142001  -2.6229581  -4.2703911   3.5573586
  -0.1375587  -4.0478054   3.7838459  -0.1378024  -4.0476203  -0.1124509
  -0.1162983  -3.5392343   4.8915161  -0.1166300  -3.5377508  -1.2197103
  -2.6001129  -4.2339211  -0.9810406  -3.1836755  -5.1521262   0.4464911
  -2.5993092  -4.2332567   4.6525757  -3.1833788  -5.1519781   3.2254427
   4.6981368  -2.1488402   1.8353715   3.8730362  -3.4478512   1.8353576
   4.2765666  -1.4853295   0.6085307   4.2765090  -1.4853704   3.0622265
   5.7785047  -2.3338920   1.8353582   4.4997274  -4.3471271   1.8353243
   3.0933012  -3.3482507   0.6084788   3.0933378  -3.3482292   3.0622608
   2.3198706  -4.4804259   0.1137444   2.3199372  -4.4802624   3.5571573
   3.3694650  -2.2164141   3.7837026   3.3691377  -2.2161746  -0.1126255
   2.9399079  -1.9440608   4.8915274   2.9382765  -1.9429335  -1.2199368
   2.2995500  -4.4416634  -0.9814830   2.8034690  -5.4063779   0.4456973
   2.2995553  -4.4411412   4.6523709   2.8035294  -5.4062583   3.2253361
   4.1482968   2.9225248   1.8355559   4.8578898   1.5568435   1.8355003
   3.3638728   2.8904764   0.6079580   3.3637403   2.8903164   3.0630749
   4.8505880   3.7640864   1.8356180   5.9502231   1.6476557   1.8355263
   4.3805512   0.9319524   0.6087251   4.3804102   0.9318685   3.0621834
   4.9730633  -0.3043960   0.1135243   4.9727928  -0.3044622   3.5574052
   3.5391586   1.7379552   3.7834924   3.5390402   1.7379971  -0.1122297
   3.0865466   1.5019035   4.8904655   3.0851544   1.5014177  -1.2187810
   4.9287899  -0.3024170  -0.9816811   6.0169147  -0.3493307   0.4448703
   4.9280805  -0.3025355   4.6525928   6.0167166  -0.3493568   3.2262817
   0.0000881   0.0022043   7.3599862  -0.9140255  -0.5765817   7.1794233
   0.8629383  -0.6505095   7.1792815   0.0340352   0.8200913   6.6298441
   0.0233669   0.5600083   8.7965518   0.9307350   1.1605837   8.9350653
  -0.0231946  -0.5607561   9.8574454  -0.8309965   1.2338392   8.9351455
  -0.9304814  -1.1613541   9.7185006   0.8311663  -1.2345217   9.7185219
  -0.0000719  -0.0034990  11.2944218  -0.8629540   0.6493080  11.4746275
  -0.0341071  -0.8217222  12.0241817   0.9140017   0.5754392  11.4746792
  -0.0000910   0.0002550  -6.0046133  -0.0002265   0.0001009  -9.0000703
   0.0001685   2.2000035 -11.2000237
