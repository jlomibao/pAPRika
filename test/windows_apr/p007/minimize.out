
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:23:39

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p007
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   631597 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9792E+02     1.0085E+01     2.4829E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -48.2797  EEL     =     1326.5230  EGB        =     -127.8627
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -797.9172
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1222E+02     4.6855E-01     1.6091E+00     H10        35

 BOND    =        1.1155  ANGLE   =      107.7359  DIHED      =       51.4826
 VDWAALS =      -48.9522  EEL     =     1337.1569  EGB        =     -133.8476
 1-4 VDW =        5.2179  1-4 EEL =    -2132.1373  RESTRAINT  =        0.0126
 EAMBER  =     -812.2283
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1222E+02     4.6854E-01     1.6091E+00     H10        35

 BOND    =        1.1157  ANGLE   =      107.7359  DIHED      =       51.4824
 VDWAALS =      -48.9526  EEL     =     1337.1619  EGB        =     -133.8488
 1-4 VDW =        5.2182  1-4 EEL =    -2132.1408  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1222E+02     4.6853E-01     1.6092E+00     H10        35

 BOND    =        1.1159  ANGLE   =      107.7360  DIHED      =       51.4821
 VDWAALS =      -48.9531  EEL     =     1337.1674  EGB        =     -133.8501
 1-4 VDW =        5.2184  1-4 EEL =    -2132.1448  RESTRAINT  =        0.0126
 EAMBER  =     -812.2283
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1222E+02     4.6852E-01     1.6092E+00     H10        35

 BOND    =        1.1161  ANGLE   =      107.7362  DIHED      =       51.4819
 VDWAALS =      -48.9536  EEL     =     1337.1728  EGB        =     -133.8515
 1-4 VDW =        5.2187  1-4 EEL =    -2132.1488  RESTRAINT  =        0.0126
 EAMBER  =     -812.2283
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1222E+02     4.6851E-01     1.6092E+00     H10        35

 BOND    =        1.1163  ANGLE   =      107.7362  DIHED      =       51.4817
 VDWAALS =      -48.9541  EEL     =     1337.1779  EGB        =     -133.8526
 1-4 VDW =        5.2189  1-4 EEL =    -2132.1526  RESTRAINT  =        0.0126
 EAMBER  =     -812.2283
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1222E+02     4.6851E-01     1.6092E+00     H10        35

 BOND    =        1.1165  ANGLE   =      107.7363  DIHED      =       51.4814
 VDWAALS =      -48.9547  EEL     =     1337.1836  EGB        =     -133.8540
 1-4 VDW =        5.2192  1-4 EEL =    -2132.1566  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1222E+02     4.6850E-01     1.6092E+00     H10        35

 BOND    =        1.1167  ANGLE   =      107.7364  DIHED      =       51.4812
 VDWAALS =      -48.9551  EEL     =     1337.1886  EGB        =     -133.8552
 1-4 VDW =        5.2194  1-4 EEL =    -2132.1602  RESTRAINT  =        0.0126
 EAMBER  =     -812.2283
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1222E+02     4.6849E-01     1.6092E+00     H10        35

 BOND    =        1.1169  ANGLE   =      107.7365  DIHED      =       51.4810
 VDWAALS =      -48.9557  EEL     =     1337.1941  EGB        =     -133.8565
 1-4 VDW =        5.2197  1-4 EEL =    -2132.1641  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1222E+02     4.6848E-01     1.6092E+00     H10        35

 BOND    =        1.1171  ANGLE   =      107.7366  DIHED      =       51.4808
 VDWAALS =      -48.9562  EEL     =     1337.1995  EGB        =     -133.8578
 1-4 VDW =        5.2199  1-4 EEL =    -2132.1681  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1222E+02     4.6847E-01     1.6092E+00     H10        35

 BOND    =        1.1172  ANGLE   =      107.7367  DIHED      =       51.4805
 VDWAALS =      -48.9567  EEL     =     1337.2047  EGB        =     -133.8590
 1-4 VDW =        5.2202  1-4 EEL =    -2132.1718  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1222E+02     4.6847E-01     1.6092E+00     H10        35

 BOND    =        1.1172  ANGLE   =      107.7367  DIHED      =       51.4805
 VDWAALS =      -48.9567  EEL     =     1337.2047  EGB        =     -133.8590
 1-4 VDW =        5.2202  1-4 EEL =    -2132.1718  RESTRAINT  =        0.0126
 EAMBER  =     -812.2282
 NMR restraints: Bond =    0.011   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.306    9.302    0.005    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.404  147.398    0.007    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.472   49.483    0.011    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.017   62.003    0.015    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.194 -146.178    0.016    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.555  -72.548    0.007    0.000 t
  DUM  DM1    3 --  C    BUT    2:   13.365   13.412    0.047    0.011 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.994  180.000    0.006    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.909  180.000    0.091    0.000 a
                                       Total distance penalty:      0.011
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.471
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.17 (21.53% of Gen B)
|                Calc gb diag               3.93 (38.99% of Gen B)
|                Calc gb off-diag           3.98 (39.46% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.08 (100.0% of Nonbo)
|          Nonbond force             10.08 (90.35% of Force)
|          Bond/Angle/Dihedral        1.04 ( 9.35% of Force)
|          Other                      0.03 ( 0.30% of Force)
|       Force time                11.16 (100.0% of Runmd)
|    Runmd Time                11.16 (99.81% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.18 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:23:39.628  on 04/10/2019
|           Setup done at 18:23:39.633  on 04/10/2019
|           Run   done at 18:23:50.810  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
